app/javascript/tests/history/HistoryList.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { shallow } from 'enzyme';
+import { Button, Col, Collapse, Row } from 'react-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import _ from 'lodash';
+import HistoryList from '../../components/history/HistoryList.js'
+import { mockPatient1} from '../mocks/mockPatients'
+import { mockHistory1, mockHistory2 } from '../mocks/mockHistories'
+import { nameFormatter, formatDate } from '../util.js'
+
+const goToMock = jest.fn();
+const authyToken = 'Q1z4yZXLdN+tZod6dBSIlMbZ3yWAUFdY44U06QWffEP76nx1WGMHIz8rYxEUZsl9sspS3ePF2ZNmSue8wFpJGg==';
+
+beforeEach(() => {
+  window.BASE_PATH = ""
+})
+
+describe('HistoryList', () => {
+  it('Properly renders all history components', () => {
+    const wrapper = shallow(<HistoryList patient_id={mockPatient1.id} histories={[]} authenticity_token={authyToken} 
+      history_types={{enrollment: "Enrollment", comment: "Comment"}}/>);
+    expect(wrapper.find('#histories').exists()).toBeTruthy();
+    expect(wrapper.find('#histories').find('span').text()).toEqual('History ');
+    expect(wrapper.find('#histories').find('textarea').exists()).toBeTruthy();
+  });
+});
```

Comment:
This component has additional functionality that needs to be tested.  I would like to see tests for
- The two selects call the right functions and properly update the state
- The list is properly filtered
- Pagination
- Changing text in the input properly updates state
- The add comment button is disabled when it should be
- Clicking the add comment button calls the right function

Reply:
added here: https://github.com/SaraAlert/SaraAlert/pull/793/commits/336f7f2565ad7b9798767f02dd44c3442b64135d